### DISCUSSION TITLE
Change project variables in Editor settings

### DISCUSSION THREAD

**Discussion by juhamust** [motion-canvas/motion-canvas#973] [Ideas]
It would be handy if one could modify [project variables](https://motioncanvas.io/docs/project-variables) values in the Motion Canvas -editor, for example via the Settings tab.

![image](https://github.com/motion-canvas/motion-canvas/assets/781002/02a8b256-a3c1-418a-84da-b3eb6c902986)

Then I could easily toggle for example debug mode off and on again, without changing the code. 

![image](https://github.com/motion-canvas/motion-canvas/assets/781002/d71d4441-87fe-4f8b-91d2-2a81753920e6)

Value could be stored in `.meta` file?

**Comment by aarthificial** [comment]
This feature will most likely be added along-side [editable signals](https://github.com/motion-canvas/motion-canvas/issues/170)
